src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceAmendmentTests.cs

```diff
@@ -632,23 +634,24 @@ private Mock<IUserService> UserServiceMock()
         private ReleaseService BuildReleaseService(
             ContentDbContext contentDbContext,
             StatisticsDbContext statisticsDbContext,
-            IPublishingService publishingService = null,
-            IPersistenceHelper<ContentDbContext> persistenceHelper = null,
-            IUserService userService = null,
-            IReleaseRepository repository = null,
-            IReleaseFileRepository releaseFileRepository = null,
-            ISubjectRepository subjectRepository = null,
-            IReleaseDataFileService releaseDataFileService = null,
-            IReleaseFileService releaseFileService = null,
-            IDataImportService dataImportService = null,
-            IFootnoteService footnoteService = null,
-            IDataBlockService dataBlockService = null,
-            IReleaseSubjectRepository releaseSubjectRepository = null,
-            IContentService contentService = null,
-            IReleaseChecklistService releaseChecklistService = null,
-            IGuidGenerator guidGenerator = null)
+            IPublishingService? publishingService = null,
```

Comment:
Wow the ReleaseService is definitely too big :joy:

Reply:
I'll create a ticket to address it

Reply:
Turns out there is already a ticket: EES-212.